README.md

```diff
@@ -21,4 +21,6 @@ Embody is a dungeon crawler where you play as a ghost that can possess any enemy
 - [Interface Sounds Starter Pack by p0ss](https://opengameart.org/content/interface-sounds-starter-pack)
 - [Menu Selection Click by NenadSimic](https://opengameart.org/content/menu-selection-click)
 - [Melee sounds by remaxim](https://opengameart.org/content/3-melee-sounds)
-- [Devil phase two audio by daf024](https://freesound.org/people/daf024/sounds/84615/)
\ No newline at end of file
+- [Devil phase two audio by daf024](https://freesound.org/people/daf024/sounds/84615/)
+- [Devil boss death audio titled Jesus Lastra - Cries From Hell (https://opengameart.org/content/dark-ambiance-cries-from-hell)
```

Comment:
Don't forget the closing bracket after "Hell" or it won't format properly :)

Reply:
Done :)